content/_landing/index._zh.md

```diff
@@ -11,22 +11,22 @@ three_col:
   col_class1: individuals
   col1: >-
     使用一个账号和密码来安全、私密地访问参与这个项目的政府机构
-  
+
 
     [了解 Login.gov](/zh/what-is-login/){:class="why-more-info"}
   subheading2: 机构合作伙伴
   col_class2: partners
   col2: >-
     用数字安全和用户体验的最高标准来保护你的用户信息Login.gov 管理软件开发、安全运营、以及客户支持，使你没必要做这些事情。
-  
+
     [成为合作伙伴](/partners/){:class="why-more-info"}
   subheading3: 机构开发商
   col_class3: developers
   col3: >-
     程序员资源、实时支持、以及现代化工具来帮助你执行和部署自己在 Login.gov 的应用程序,
 
 
-    [参见开发人员指南]((https://developers.login.gov/){:class="why-more-info"}
+    [参见开发人员指南](https://developers.login.gov/){:class="why-more-info"}
```

Comment:
We have a spec in the dev docs repo:

https://github.com/GSA-TTS/identity-dev-docs/blob/ab0d27e846cfecfa7a1d8d08b959b61aed3d31b3/spec/page.md_spec.rb#L15

I think we used to have it here too, might be worth bringing it back?

Reply:
Good call, added in 7051c53 and verified that it fails on `main`:

```
Failures:

  1) all pages zh/what-is-login/index.html does not have broken Markdown links
     Failure/Error: expect(doc.to_s).to_not include(')]')
       expected "<!DOCTYPE html>\n<html lang=\"zh\">\n\n  <head>\n<meta http-equiv=\"Content-Type\" content=\"text/ht...oter>\n\n\n    \n\n\n  \n<script src=\"/assets/js/main.js\"></script>\n\n\n\n  </body>\n\n</html>\n" not to include ")]"
     # ./spec/all_pages_spec.rb:44:in `block (4 levels) in <top (required)>'
```